frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx

```diff
@@ -376,8 +376,8 @@ export const UploadPVCForm: React.FC<UploadPVCFormProps> = ({
               inputID="request-size-input"
             />
             <p className="help-block" id="request-size-help">
-              Ensure your PVC size covers the requirements of the uncompressed image and any other
-              space requirements
+              Ensure that the Persistent Volume Claim is larger than the uncompressed image and meets any other
```

Comment:
I prefer `covers the requirements` it's less open ended. Saying `is larger` might beg the question : How much larger? I am also a little torn on PVC vs Persistent Volume Claim - at this point the user likely understands what a PVC is and doesn't need it in full form.